app/jobs/close_patients_job.rb

```diff
@@ -30,8 +30,22 @@ def perform(*_args)
         patient[:monitoring_reason] = 'Completed Monitoring (system)'
       end
 
-      # Send closed email to patient if they are a reporter
-      PatientMailer.closed_email(patient).deliver_later if patient.save! && patient.email.present? && patient.self_reporter_or_proxy?
+      # Send closed email or SMS to patient if they are a reporter
+      if patient.save! && patient.self_reporter_or_proxy?
+        contact_method = patient.preferred_contact_method&.downcase
+        if ['sms texted weblink', 'sms text-message'].include? contact_method
+          PatientMailer.closed_sms(patient).deliver_later(wait_until: patient.time_to_contact_next)
+        elsif contact_method == 'e-mailed web link'
+          PatientMailer.closed_email(patient).deliver_later(wait_until: patient.time_to_contact_next)
+        else
+          History.record_automatically_closed(
+            patient: patient,
+            comment: 'The system was unable to send a closed notification to this patient via '\
+                     "#{patient.preferred_contact_method}, because the preferred contact method is not supported"\
+                     'for closed notifications.'
```

Comment:
@holmesie does this text need to be approved?

Reply:
Yes, run it by PO

Reply:
requested language was “The system was unable to send a monitoring complete message to this monitoree because their preferred contact method, Telephone Call, is not supported for this message type.”